### ISSUE TITLE
FIWARE Interoperability: Timestamps should be of type DateTime (ISO8601)

### ISSUE THREAD

**Issue by jmcanterafonseca** [OpenMTC/OpenMTC#15]
For the sake of completeness and semantics instead of

```
{
  "type": "Float",
  "value": 20.569353172,
  "metadata": {
    "bn": {
      "type": "String",
      "value": "urn:dev:s300th:1"
    },
    "timestamp": {
      "type": "String",
      "value": "1527093220.104"
    },
    "unit": {
      "type": "String",
      "value": "Cel"
    }
  }
}
```

It should be

```
{
  "type": "Float",
  "value": 20.569353172,
  "metadata": {
    "bn": {
      "type": "String",
      "value": "urn:dev:s300th:1"
    },
    "timestamp": {
      "type": "DateTime",
      "value": "2019-01-08T12:05:00Z"
    },
    "unit": {
      "type": "String",
      "value": "Cel"
    }
  }
}
```

**Comment by rst-fokus** [comment]
Hello Jose,

We are using in OpenMTC for storing data a format similar to SenML, but a little bit adapted. And so we are mapping some of the data into metadata in the context broker.

Is there a specific format specified in FIWARE for the metadata? Then I will update the mapping. Date transformation can be done quite effortlessly.

Best,
Ronald

**Comment by jmcanterafonseca** [comment]
for timestamp metadata in FIWARE, it is usually used the `timestamp` field, you can see several examples in the FIWARE Data Models project. The value shall be encoded in ISO8601 format. 

you can see an example at

https://github.com/Fiware/dataModels/blob/master/specs/Parking/ParkingSpot/example-normalized.json

**Comment by aor-fokus** [comment]
Hello @jmcanterafonseca ,

We fixed this some time ago with commit c3ac4f9dfbc313962ed894fd1239c000f04abd0e

I would say its solved now and close this Issue.
Thanks for your note.
